source/AI.cpp

```diff
@@ -1836,7 +1836,7 @@ void AI::DoCloak(Ship &ship, Command &command)
 		
 		// If this ship has started cloaking, it must get at least 40% repaired
 		// or 40% farther away before it begins decloaking again.
-		double hysteresis = ship.Cloaking() ? 1.4 : 1.;
+		double hysteresis = ship.IsCloaking() ? 1.4 : 1.;
```

Comment:
Would it work to just use `ship.Commands().Has(Command::CLOAK)` instead of adding a whole new function and member variable to Ship?